@kiva/kv-components/src/vue/KvWwwHeader/KvHeaderLinkBar.vue

```diff
@@ -92,7 +92,7 @@
 			data-testid="header-basket"
 		>
 			<kv-icon-bag
-				class="tw-w-3 tw-h-3 md:tw-w-3.5 md:tw-h-3.5 tw-pointer-events-none"
+				class="tw-w-3 tw-h-3 md:tw-w-3.5 md:tw-h-3.5 tw-pointer-events-none tw-text-action"
```

Comment:
I think this should be only on hover, that's the current monolith behavior  https://www.development.kiva.org/sitemap?setuiab=home_page.b

Reply:
Oh you know what, different monolith pages have different behaviors in the new nav... Let me check with product.

Reply:
Okay yeah we want this icon green.